python/pyspark/pandas/tests/test_dataframe_conversion.py

```diff
@@ -90,7 +90,6 @@ def get_excel_dfs(pandas_on_spark_location, pandas_location):
             "expected": pd.read_excel(pandas_location, index_col=0),
         }
 
-    @unittest.skip("openpyxl")
     def test_to_excel(self):
```

Comment:
What about this case for pypy3? Will it affect developers run tests?

(I remember we run pypy3 and python3.9 in CI....[but not](https://github.com/itholic/spark/runs/8240425530?check_suite_focus=true#step:11:1174) running, pls let me know if I missed something history changes...)

Reply:
Actually PyPy is not tested with Pandas API on Spark per https://github.com/apache/spark/blob/master/dev/sparktestsupport/modules.py#L663-L667 because pyarrow, etc are not available with PyPy. Should probably docunent it somewhere but let's do that separately.

Reply:
Thanks, good to know!